packages/app/src/components/pages/VerifyEmail/index.js

```diff
@@ -1,11 +1,19 @@
-import React from 'react'
+import React, { useEffect } from 'react'
+import utilsAPI from '../../../utils/API/nominationsAPI.js';
+
+const VerifyEmail = (props) => {
+  
```

Comment:
Nitpick: I think this newline is unnecessary.

Reply:
you got it